api/v1alpha1/validation.go

```diff
@@ -236,7 +236,6 @@ func (r EnrollmentRequestApproval) Validate() []error {
 
 func (r CertificateSigningRequest) Validate() []error {
 	allErrs := []error{}
-	allErrs = append(allErrs, validation.ValidateResourceName(r.Metadata.Name)...)
```

Comment:
Please leave in. All of our API resources conform to this for consistency and compatibility with K8s.

Reply:
Okay no problem.